ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/site1-sno-du.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     name: "sshPrvKey"
   clusters:
   - clusterName: "site-sno-du-1"
+    biosConfigMapRef:
+      name: "testSiteConfig/hp.profile"
```

Comment:
For test data we should keep it generic and not reference a specific hardware vendor. Maybe testHW.profile?

Reply:
Yeah.